flyteidl/protos/flyteidl/core/execution.proto

```diff
@@ -50,10 +50,11 @@ message TaskExecution{
         SUCCEEDED = 3;
         ABORTED = 4;
         FAILED = 5;
+        RETRYABLE_FAILED = 6;
```

Comment:
We shouldn't change the tag number. https://protobuf.dev/best-practices/dos-donts/
```suggestion
        RETRYABLE_FAILED = 8;
```